tests/phpunit/Elastic/Indexer/IndexerTest.php

```diff
@@ -81,7 +82,6 @@ public function testCreate() {
 
 		$expected = [
 			'index' => '_index_abc',
-			'type' => 'data',
```

Comment:
This change is not obvious to me.

Reply:
This is to account for https://github.com/SemanticMediaWiki/SemanticMediaWiki/pull/5451/commits/f020de1b52ff7ff833fdf0394a3ae81ee40139e2 which removed these types.